exchanges/okx/okx_test.go

```diff
@@ -942,727 +1147,761 @@ func TestCreateQuote(t *testing.T) {
 }
 
 func TestGetPublicBorrowInfo(t *testing.T) {
 	t.Parallel()
-	if _, err := ok.GetPublicBorrowInfo(contextGenerate(), ""); err != nil {
-		t.Error("Okx GetPublicBorrowInfo() error", err)
-	}
-	if _, err := ok.GetPublicBorrowInfo(context.Background(), "USDT"); err != nil {
-		t.Error("Okx GetPublicBorrowInfo() error", err)
-	}
+	result, err := ok.GetPublicBorrowInfo(contextGenerate(), currency.EMPTYCODE)
+	assert.NoError(t, err)
+	assert.NotNil(t, result)
+
+	result, err = ok.GetPublicBorrowInfo(context.Background(), currency.USDT)
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetPublicBorrowHistory(t *testing.T) {
 	t.Parallel()
-	if _, err := ok.GetPublicBorrowHistory(context.Background(), "USDT", time.Time{}, time.Time{}, 1); err != nil {
-		t.Error("Okx GetPublicBorrowHistory() error", err)
-	}
+	result, err := ok.GetPublicBorrowHistory(context.Background(), currency.USDT, time.Time{}, time.Time{}, 1)
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetConvertCurrencies(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ok)
-
-	if _, err := ok.GetConvertCurrencies(contextGenerate()); err != nil {
-		t.Error("Okx GetConvertCurrencies() error", err)
-	}
+	result, err := ok.GetConvertCurrencies(contextGenerate())
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestGetConvertCurrencyPair(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ok)
-
-	if _, err := ok.GetConvertCurrencyPair(contextGenerate(), "USDT", "BTC"); err != nil {
-		t.Error("Okx GetConvertCurrencyPair() error", err)
-	}
+	result, err := ok.GetConvertCurrencyPair(contextGenerate(), currency.USDT, currency.BTC)
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestEstimateQuote(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ok, canManipulateRealOrders)
 
-	if _, err := ok.EstimateQuote(contextGenerate(), &EstimateQuoteRequestInput{
-		BaseCurrency:  "BTC",
-		QuoteCurrency: "USDT",
+	result, err := ok.EstimateQuote(contextGenerate(), &EstimateQuoteRequestInput{
+		BaseCurrency:  currency.BTC,
+		QuoteCurrency: currency.USDT,
 		Side:          "sell",
 		RfqAmount:     30,
 		RfqSzCurrency: "USDT",
-	}); err != nil {
-		t.Error("Okx EstimateQuote() error", err)
-	}
+	})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
 }
 
 func TestConvertTrade(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ok, canManipulateRealOrders)
 
-	if _, err := ok.ConvertTrade(contextGenerate(), &ConvertTradeInput{
+	result, err := ok.ConvertTrade(contextGenerate(), &ConvertTradeInput{
 		BaseCurrency:  "BTC",
 		QuoteCurrency: "USDT",
 		Side:          "Buy",
 		Size:          2,
-		SizeCurrency:  "USDT",
+		SizeCurrency:  currency.USDT,
 		QuoteID:       "quoterETH-USDT16461885104612381",
-	}); err != nil {
-		t.Error("Okx ConvertTrade() error", err)
-	}
+	})
+	require.NoError(t, err)
```

Comment:
Just check this one against the API docs. This could just be an error message suggesting the quoteid does not exist, or that its a bad param:
```
    okx_test.go:1482: 
        	Error Trace:	/Users/WIF/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/okx/okx_test.go:1482
        	Error:      	Received unexpected error:
        	            	Okx unsuccessful HTTP status code: 400 raw response: {"code":"51000","data":[],"msg":"Invalid parameter quoteId"}
        	Test:       	TestConvertTrade
```